### DISCUSSION TITLE
Changes to starting night capture

### DISCUSSION THREAD

**Discussion by SteveWasT** [aaronwmorris/indi-allsky#982] [General]
I've updated to the latest code and this has caused a slight issue.

The images for night-time captures are starting earlier then before, now starts before sunset.

I've noticed that astropanel is giving a wrong time for local sunset (currently it should be around 18:30 but astropanel is suggesting 15:22). Whether that is related I don't know.

I have the sun altitude at -6 in config and have checked my location settings.

I'll have a bash at debugging capture.py tomorrow but if this is already a known problem/solution please let me know, by debugging is slow!

Thanks

Steve

**Comment by aaronwmorris** [comment]
Is your timezone set correctly on your system?

**Reply by SteveWasT** [reply]
Yes currently showing as:
$ timedatectl
               Local time: Thu 2023-10-12 14:36:03 BST
           Universal time: Thu 2023-10-12 13:36:03 UTC
                 RTC time: n/a
                Time zone: Europe/London (BST, +0100)

NIGHT_SUN_ALT_DEG is set as -6 as per online instructions. If I follow the code correctly the boolean self.night gets set to true when the sun altitude is less than the config setting (-6 expressed in radians).

I'll dig a little deeper for the cause.

**Comment by aaronwmorris** [comment]
What is your longitude/latitude?  If you are not comfortable posting here, you can email them to me at [aaron@aarmor.net](mailto:aaron@aarmor.net)

**Reply by SteveWasT** [reply]
Lat -0.165
Long 51.8827
Elevation 100

First image saved at 15:44 today (no daytime images otherwise)

Thanks
Steve

**Reply by aaronwmorris** [reply]
I am not sure of your actual location, but is it possible you have your latitude and longitude backwards?  That latitude and longitude is in the Indian Ocean off the coast of Somalia.

**Reply by SteveWasT** [reply]
Yep, I must have had a senior moment and blindly copied them from notepad which were listed as:
Long -0.165
Lat 51.8827
Elevation 100

And pasted them in that order. I only had to edit them again as I'd moved the system to a bigger sdcard and re-installed from a saved image file.
Thanks
